config/templates/hive-csv-template.yaml

```diff
@@ -4,56 +4,108 @@ metadata:
   name: hive-operator-0.0.1
   namespace: placeholder
   annotations:
-    categories: A list of comma separated categories that your operator falls under.
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
     certified: "false"
     description: OpenShift cluster provisioning and management at scale.
-    containerImage: quay.io/dgoodwin/hive:latest
+    containerImage: quay.io/openshift-hive/hive:v0.0.1
     createdAt: "2019-02-25T14:29:00Z"
-    support: Devan Goodwin
+    repository: https://github.com/openshift/hive
+    support: Hive Team
+    alm-examples: |-
+      [{"apiVersion":"hive.openshift.io/v1","kind":"HiveConfig","metadata":{"name":"hive"},"spec":{"managedDomains":[{"aws":{"credentialsSecretRef":{"name":"my-route53-creds"}},"domains":["my-base-domain.example.com"]}]}}]
```

Comment:
Out of curiosity, does the example have to be json? Is the intention to put it all on one line so that it does not overwhelm the rest of the yaml?

Reply:
yeah, oddly, it seemed yaml wasn't working, and all the other examples I found used json.

I blindly ran yaml2json and cried happy tears at the mere sight of my old buddy json. It can totally be on multiple lines if you prefer.

Reply:
No, don't change it.